docs/proposals/prop-0001-proposals.md

```diff
@@ -51,16 +51,16 @@ proposal.
 
 Proposals are discussed on the anonlink mailing list. The possible paths of the status of a proposal are:
 
-![](./prop-1/resolution-process.png)
+![](prop-0001/resolution-process.png)
```

Comment:
The folder name needs to be updated to include the short name too, or will the folders be named without the short-name?

Reply:
Yes - the folder name has already been updated to include `0` padded numbers too.